### PR TITLE
[stftpitchshift] Update to 1.4.1

### PR DESCRIPTION
**Pull request by jurihock**
**Describe the pull request**

- #### What does your PR fix?
  update stftpitchshift to v1.4.1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all, yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/stftpitchshift/portfile.cmake [hunk 1]
```diff
@@ -14,6 +14,17 @@ vcpkg_cmake_configure(
...
 
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/stftpitchshift" RENAME copyright)
+vcpkg_cmake_config_fixup(
+  PACKAGE_NAME "stftpitchshift"
+  CONFIG_PATH "lib/cmake/stftpitchshift"
+)
+
+file(
+  INSTALL "${SOURCE_PATH}/LICENSE"
+  DESTINATION "${CURRENT_PACKAGES_DIR}/share/stftpitchshift"
```
Comment (Cheney-W) [inline ports/stftpitchshift/portfile.cmake:24]:
```suggestion
  DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"
```

**Comment by jurihock** [comment]
@Cheney-W My original commit 1513d67 was successful, but after your suggestion 8c82315 there is a build error. Please troubleshoot...

**Comment by Cheney-W** [comment]
• .\vcpkg.exe x-add-version {port name} --overwrite-version
	• git add .
	• git commit -m "update json"
	• git push

**Comment by jurihock** [comment]
Yeah, the overhead of manual vcpkg internal fixes (like "update version database" etc.) for just a single commit in the pull request is ridiculous! Instead of spending own free time efficiently to work on open source projects and do something useful...

File: ports/stftpitchshift/portfile.cmake [hunk 2]
```diff
@@ -14,6 +14,17 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/stftpitchshift" RENAME copyright)
+vcpkg_cmake_config_fixup(
+  PACKAGE_NAME "${PORT}"
```
Comment (Cheney-W) [inline ports/stftpitchshift/portfile.cmake:18]: Thie sentence could be removed, PACKAGE_NAME defaults to ${PORT}.

**Comment by jurihock** [comment]
So what else?

**Review by Cheney-W — APPROVED** [review]

**Comment by Cheney-W** [comment]
Thanks for your prompt reply, now just waiting for merge.

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**